tests/next_tests/unit_tests/type_system_tests/dummy_package/__init__.py

```diff
@@ -0,0 +1,9 @@
+# GT4Py - GridTools Framework
```

Comment:
let's maybe move this to the top level `test/next_tests` as it's now used in both integration_tests and unit_tests or duplicate with only the relevant stuff inside.